Doc/library/tempfile.rst

```diff
@@ -162,7 +162,9 @@ The module defines the following user-callable items:
    by child processes.
 
    Unlike :func:`TemporaryFile`, the user of :func:`mkstemp` is responsible
-   for deleting the temporary file when done with it.
+   for deleting the temporary file when done with it. If a large number of file
+   descriptors are created, your program may run into a per-process limit and raise an :exc:`OSError`.
+   You can avoid this issue by closing file descriptors with :func:`os.close`.
```

Comment:
I would just say something like "the user of :func:`mskstemp` is responsible for deleting the temporary file and closing the file descriptor". Lots of things can cause you to run out of FDs, this isn't the place to describe the exact behavior.

Reply:
I agree.